Yosemite/Yosemite/Model/Enums/CardPresentPaymentsOnboardingState.swift

```diff
@@ -6,7 +6,7 @@ public enum CardPresentPaymentOnboardingState: Equatable {
 
     /// All the requirements are met and the feature is ready to use
     ///
-    case completed
+    case completed(plugin: CardPresentPaymentsPlugins)
```

Comment:
Did you consider storing the `PaymentGatewayAccount` here instead? Since `updateAccounts` is already trying to refresh accounts and `accountChecks` gets the current account, I think it shouldn't be too hard to include it, and then everything downstream can immediately know which account to use. Not a blocker for this PR, but since we're adding something to `completed`, the account might be more useful than the plugin.

Reply:
I mostly passed plugin because it was the associated value present for other cases - but you're right - passing the account would likely be more useful. I'll open an issue for that so this PR can move forward.